bigbluebutton-html5/imports/api/notifications/server/handlers/breakoutTimeUpdated.js

```diff
@@ -0,0 +1,19 @@
+import { check } from 'meteor/check';
+import emitter from '../emitter';
+
+export default function ({ body }, meetingId) {
+  const { timeInMinutes } = body;
+
+  check(meetingId, String);
+  check(timeInMinutes, Number);
+
+  const mainRoomMsgKey = 'app.meeting.alertBreakoutTimeExtended.mainRoom';
+  const breakoutRoomMsgKey = 'app.meeting.alertBreakoutTimeExtended.breakout';
+
+  emitter.emit('notification', meetingId, {
```

Comment:
please break the .emit part in two (one for main room and another for breakout room), so it becomes more generic and we can reuse it later (one message key = one notification)